pkg/schedule/windowed_schedule_test.go

```diff
@@ -14,11 +14,55 @@ import (
 func TestWindowedSchedule(t *testing.T) {
 	log.SetLevel(log.DebugLevel)
 	Convey("Windowed Schedule", t, func() {
+		Convey("nominal window without misses", func() {
+			startWait := time.Millisecond * 50
+			windowSize := time.Millisecond * 200
+			interval := time.Millisecond * 10
+
+			start := time.Now().Add(startWait)
+			stop := time.Now().Add(startWait + windowSize)
+			w := NewWindowedSchedule(
+				interval,
+				&start,
+				&stop,
+			)
+
+			err := w.Validate()
+			So(err, ShouldBeNil)
+
+			var r []Response
+			last := *new(time.Time)
+
+			state := Active
+			before := time.Now()
+			for state == Active {
+				r1 := w.Wait(last)
+				state = r1.State()
+				last = time.Now()
+				r = append(r, r1)
+			}
+			// we should have either 21 or 23 minus 0 missed
```

Comment:
"We should have either 21 or 23?", is an odd/inaccurate comment given we are checking for a value between 20 and 22.  It should probably read, "we expect a range between x and y".

Also, this test is flaky >50% of the time on my system.

![img](https://www.evernote.com/l/AA0QQE8CRghMlIWUfRPJ8yMJDYYysWgjoUEB/image.png)